app/controllers/graphql_controller.rb

```diff
@@ -15,6 +14,21 @@ def execute
   end
 
   private
+  def current_user
+    credential = setup_credentials
+
+    return unless credential
+    credential.current_user_id
+  end
+
+  def setup_credentials
+    authorization = request.headers['HTTP_AUTHORIZATION']
+    match = /\ABearer (.*)\Z/.match(authorization)
```

Comment:
Copied your (@camallen) code pretty much directly for this. Is the match searching for a specific line in the JWT?

Reply:
> Is the match searching for a specific line in the JWT?

this match will just pull out the whole JWT form the Authorization header minus the `Bearer ` string prefix.